src/App.scss

```diff
@@ -1,23 +1,74 @@
-.Sidebar {
-  overflow: hidden;
-  opacity: 0;
-  transition-property: max-width, opacity;
-  transition-duration: 0.5s;
-  transition-timing-function: ease-in-out;
-
-  @media (min-width: 769px) {
-    max-width: 0;
+* {
+  box-sizing: border-box;
+}
+
+body {
+  min-height: 100vh;
+  background-color: #e9e6f2;
+  padding: 20px;
+  color: #4d457b;
+  font-family: "Roboto", sans-serif;
```

Comment:
What if the user does not have Roboto font or by any reason he cannot be able to download it? We should be able to provide a fallback font in it, reducing the impact of the error on the user experience.

It is important, so consider call the design team to discuss this topic in more depth.